docs/user-guide/src/capm3/installation_guide.md

```diff
@@ -0,0 +1,29 @@
+# Install Cluster-api-provider-metal3
+
+You can either use clusterctl (recommended) to install Metal³ infra provider
+or kustomize and directly install manifests yourself. Please keep in mind that
+Baremetal Operator and Ironic are decoupled from CAPM3 and will not be installed
+when the provider is initialized. As such, you need to install them yourself and you
+can do it by following instructions in [Install Baremetal Operator](../bmo/install_baremetal_operator.md)
+and [Install Ironic](../bmo/install_ironic.md).
+
+## Prerequisites
+
+1. `clusterctl`, refer to Cluster API [book](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl) for installation instructions.
```

Comment:
What is the Prerequisites ?
Reading/knowing about clusterctl and kustomize
or installing the binaries ?
or building them from some specific branch ?

Reply:
added, thanks